libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -120,3 +135,42 @@ func getPercpuUsage(path string) ([]uint64, error) {
 	}
 	return percpuUsage, nil
 }
+
+func getPercpuUsageInModes(path string) ([]uint64, []uint64, error) {
+	percpuUsageKernelMode := []uint64{}
+	percpuUsageUserMode := []uint64{}
```

Comment:
NIT: These names are a bit long given that the function already tells you that it's getting the per-cpu usage information. But I don't mind too much.